ci/lib.sh

```diff
@@ -93,3 +94,17 @@ export OS
 # RELEASE_PATH is the destination directory for the release from the root.
 # Defaults to release
 RELEASE_PATH="${RELEASE_PATH-release}"
+
+# Symlink node_modules.asar to node_modules. VS Code needs a node_modules.asar
+# but that's just a duplicate of stuff we already have in node_modules.
+symlink_asar() {
+  if [ ! -e node_modules.asar ]; then
+    if [ "${WINDIR-}" ]; then
+      # mklink takes the link name first.
+      mklink /J node_modules.asar node_modules
```

Comment:
God bless you for trying to support windows @code-asher 

haha

Only benefit to writing scripts in javascript is that they'll work on windows too. Maybe we should consider that or adding something to our scripts to pull in something like `cygwin` to map standard commands for us easily. In fact, given the interpreter here is `bash`, they'd need `cygwin` anyway and so have a working `ln` binary for windows.

Reply:
The weird thing about `ln -s` in Windows (using MINGW64) is that when I tested it seems to just copy the directory rather than make a link. Changes to either end don't reflect on the other side.

Reply:
weird. does installing `MINGW64` also install `cygwin`?

Reply:
I think `MINGW64` is a slim fork of `cygwin` but I'm not sure. :thinking: 

What I do is use the `git` installer and it comes with `MINGW64` with `bash` and a bunch of other things.